isis/tests/Fixtures.cpp

```diff
@@ -307,14 +307,48 @@ namespace Isis {
               {31, 1}};
     poly.Create(coords);
     cube2->write(poly);
-    cube2->reopen("rw");
 
     cube3 = new Cube();
     cube3->fromIsd(tempDir.path() + "/cube3.cub", labelPath3, *isdPath3, "rw");
 
+    LineManager line(*cube1);
+    LineManager line2(*cube2);
+    LineManager line3(*cube3);
+    int pixelValue = 1;
+    for(line.begin(); !line.end(); line++) {
+      for(int i = 0; i < line.size(); i++) {
+        line[i] = (double) (pixelValue %255);
```

Comment:
Since the three cube's pixel type is real, we don't need to make sure the pixel values are between 0-255. We can keep this though if we prefer to restrict the values.